prep-javascript-dom-manipulation/index.html

```diff
@@ -13,17 +13,21 @@
     <p class="first-paragraph">
       I really like coding.
     </p>
-    <p class="secondParagraph"> I also really like to watch television and eating pancakes.
+    <p class="secondParagraph">
+      I also really like to watch television and eating pancakes.
     </p>
-    <p class="thirdParagraph"> I hope one day to meet people that also like coding, television, and pancakes.
+    <p class="thirdParagraph">
+      I hope one day to meet people that also like coding, television, and pancakes.
```

Comment:
You can delete the text content from these paragraphs. the JavaScript code will add it.